main.go

```diff
@@ -42,19 +42,19 @@ func init() {
 func main() {
 	flag.Parse()
 
-	p := project.NewProject(&project.Context{
+	p := project.NewProject(nil, &project.Context{
```

Comment:
+1. Made the above edit to fix this error: 

`cannot use project.Context literal (type *project.Context) as type project.RuntimeProject in argument to project.NewProject:
	*project.Context does not implement project.RuntimeProject (missing RemoveOrphans method)`